### DISCUSSION TITLE
acer cb3-431 braswell

### DISCUSSION THREAD

**Discussion by 21rai21** [sebanc/brunch#2054] [Q&A]
Good afternoon.
i have acer chromebook cb3-431, processor celeron n3160 braswell/
I tried to install native recovery, Rasmus, Samus, Octopus and others.
The installation goes through without errors, but during the first boot there is a black screen with the СhromeOS logo and it won’t load any further, I tried waiting from 2 hours to 24 hours.
what is missing for normal loading?
Thank you.

**Comment by amtaher550** [comment]
did you solve it?

**Comment by vc9Kev** [comment]
I’m recently find out about crunch and I got Acer Cb3-431 also, I tried many chromeos img but not of it boot until I give a try on combination latest Brunch + Fyde OS. for a result it work perfectly
chromeOS flex had no sound, rummus and others img just won’t boot. Only Fyde OS doing great. GPU , WiFi , keyboard trackpad Bluetooth all fully supported.  If you install fydeOS without brunch keyboard will not working 
![image](https://github.com/user-attachments/assets/5d1dac77-d9d2-4f97-8fe6-c882ec58c8b0)
